Podcast/FeedTableViewCell.swift

```diff
@@ -0,0 +1,460 @@
+            contextLabel.center.y = contextViewHeight / 2
+        }
+        
+        
+        mainView.frame = CGRect(x: 0, y: contextViewHeight, width: frame.width, height: mainViewHeight)
+        bottomView.frame = CGRect(x: 0, y: contextViewHeight + mainViewHeight, width: frame.width, height: bottomViewHeight)
+        
+        episodeNameLabel.frame = CGRect(x: episodeNameLabelX, y: episodeNameLabelY, width: frame.width - episodeNameLabelRightX - episodeNameLabelX, height: episodeNameLabelHeight)
+        dateTimeLabel.frame = CGRect(x: dateTimeLabelX, y: dateTimeLabelY, width: frame.width, height: dateTimeLabelHeight)
+        descriptionLabel.frame = CGRect(x: descriptionLabelX, y: descriptionLabelY, width: frame.width - descriptionLabelRightX - descriptionLabelX, height: descriptionLabelHeight)
+        tagsLabel.frame = CGRect(x: tagsLabelX, y: tagsLabelY, width: frame.width - tagsLabelRightX - tagsLabelX, height: tagsLabelHeight)
+        podcastImage.frame = CGRect(x: podcastImageX, y: podcastImageY, width: podcastImageSize, height: podcastImageSize)
+        
+        recommendedLabel.sizeToFit()
+        bookmarkButton.frame = CGRect(x: bookmarkButtonX, y: 0, width: bookmarkButtonWidth, height: bookmarkButtonHeight)
+        recommendedButton.frame = CGRect(x: recommendedButtonX, y: 0, width: recommendedButtonWidth, height: recommendedButtonHeight)
+        recommendedButton.center.y = bottomViewHeight / 2
+        bookmarkButton.center.y = bottomViewHeight / 2
+        playButton.frame = CGRect(x: playButtonX, y: 0, width: playButtonWidth, height: playButtonHeight)
+        moreButton.frame = CGRect(x: moreButtonX, y: 0, width: moreButtonWidth, height: moreButtonHeight)
+        playButton.center.y = bottomViewHeight / 2
+        moreButton.center.y = bottomViewHeight / 2
+        recommendedLabel.center.y = bottomViewHeight / 2
+        recommendedLabel.frame.origin.x = recommendedButton.frame.origin.x + buttonPadding + recommendedButton.frame.width
+        playLabel.frame = CGRect(x: playButtonX + playButtonWidth + buttonPadding, y: 0, width: 0, height: 0)
+        playLabel.sizeToFit()
+        playLabel.center.y = bottomViewHeight / 2
+        
+        feedControlButton.frame = CGRect(x: feedControlButtonX, y: 0, width: feedControlButtonWidth, height: feedControlButtonHieght)
+        feedControlButton.center.y = contextViewHeight / 2
+
+        lineSeperator.frame = CGRect(x: lineSeperatorX, y: mainViewHeight - 1, width: frame.width - 2 * lineSeperatorX, height: lineSeperatorHeight)
+        topLineSeperator.frame = CGRect(x: 0, y: 0, width: frame.width, height: lineSeperatorHeight)
+        seperator.frame = CGRect(x: 0, y: frame.height - seperatorHeight, width: frame.width, height: seperatorHeight)
+        
+        
+    }
+    
+
+    ///
+    ///Mark - Buttons
+    ///
+    
+    
+    func bookmarkButtonPress() {
+        if let card = card as? EpisodeCard {
```

Comment:
A view should not have a reference to a model. Use delegation to tell the viewcontroller responsible for holding this cell that the bookmark button was pressed. Then the viewcontroller can update the model accordingly